Bot and Telegram Bot.py

```diff
@@ -0,0 +1,108 @@
+class Bot:
+
+    def __init__(self, name):
+        self.name = name
+
+    def say_message(self, message):
+        self.message = message
+        print(self.message)
```

Comment:
I don't see the place where `self.message` is used, so it can be removed
```suggestion
        print(message)
```